pkg/tnf/reel/reel.go

```diff
@@ -206,32 +205,31 @@ func (r *Reel) Step(step *Step, handler Handler) error {
 		} else {
 			if len(results) > 0 {
 				result := results[0]
-
 				output, outputStatus := r.stripEmulatedPromptFromOutput(result.Output)
 				if outputStatus != 0 {
 					return fmt.Errorf("error executing command exit code:%d", outputStatus)
 				}
 				match, matchStatus := r.stripEmulatedPromptFromOutput(result.Match[0])
 				log.Debugf("command status: output=%s, match=%s, outputStatus=%d, matchStatus=%d", output, match, outputStatus, matchStatus)
 
-				matchIndex := strings.Index(output, match)
-				var before string
-				// special case:  the match regex may be nothing at all.
-				if matchIndex > 0 {
-					before = output[0 : matchIndex-1]
+				if result.CaseIdx != len(batcher[0].Cases())-1 {
```

Comment:
This is a good time to rename the batcher var to "batchers", since it's a slice. Also, this deserves a comment to remind the reader that we added the "empty" batcher as the last Case in the batcher, so that's why we need to filter it out.